app/components/c-footer.vue

```diff
@@ -8,7 +8,7 @@
       <hr :class="b('navigation-separator')">
       <div :class="b('social')">Folgen Sie uns auf: (F) (L) (X)</div>
       <hr :class="b('social-separator')">
-      <div :class="b('sonepar')">A Sonepar Company</div>
+      <div :class="b('sonepar')"><a href="http://www.sonepar.com/" target="_blank">A Sonepar Company</a></div>
```

Comment:
- [ ] Please create a translation for this. From now on all inline text should be handled via translations. See https://kazupon.github.io/vue-i18n/en/